src/main/java/racingcar/ResultView.java

```diff
@@ -13,8 +23,15 @@ public static void print(CarRace carRace){
     public static String getDistanceToUI(CarSnapshot carSnapshot){
         String result = "";
         for(int i=0; i < carSnapshot.getDistance(); i++){
-            result += "-";
+            result += DISTANCE;
         }
         return result;
     }
+
+    public static void printWinner(List<Car> winnerCars){
+        String winnerName = String.join(",", winnerCars.stream().map(s -> s.getName()).collect(Collectors.toList()));
+        System.out.println(winnerName + "가 최종 우승했습니다.");
```

Comment:
```suggestion
    String winnerName = winnerCars.stream()
                .map(Car::getName)
                .collect(Collectors.joining(","));
        System.out.println(winnerName + "가 최종 우승했습니다.");
```

stream 이후는 개행을 해주면 가독성이 좋아질거 같아요! 😄